setup.py

```diff
@@ -110,6 +110,7 @@
     'ipaddr>=2.1.11,<2.2.0',
     'kombu==3.0.30',
     'lxml>=3.4.1',
+    'msgpack-python==0.4.7',
```

Comment:
We can safely do a range here, `msgpack-python>0.4,<0.5` unless there's a specific reason we shouldn't. And we'll lock it down in `getsentry`.